### PR TITLE
Ui review fix

### PR DESCRIPTION
**Pull request by Deren-Liao**
Minor UI changes on UI review feedback.
https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/539 https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/541 https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/542 https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/544 https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/545 https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/546 https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/547

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/Controls/DotsProgressIndicator.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            for (int i = 0; i < DotsCount; ++i)
+            {
+                DiscreteObjectKeyFrame kf1 = new DiscreteObjectKeyFrame(
+                    i,
+                    new TimeSpan(0, 0, 0, 0, framePoint));
+                animation.KeyFrames.Add(kf1);
+                framePoint += frameDuration;
+            }
+
+            var _storyboard = new Storyboard();
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/Controls/DotsProgressIndicator.cs]: `storyboard` not `_storyboard`

File: GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/LogsViewerToolWindowControl.xaml [hunk 2]
```diff
@@ -485,10 +486,12 @@
                     Visibility="{Binding ShowRequestStatus, Converter={StaticResource visibilityConverter}}">
 
                     <TextBlock 
-                        Padding="8,5,8,5" 
-                        Foreground="Red"
```
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/LogsViewerToolWindowControl.xaml]: Should this be blue as well?

Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/LogsViewerToolWindowControl.xaml]: For error message, using Red makes sense.

File: GoogleCloudExtension/GoogleCloudExtension/Controls/DotsProgressIndicator.cs (hunk 1 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Controls/DotsProgressIndicator.cs]: Done.

File: GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/LogsViewerToolWindowControl.xaml (hunk 2 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/LogsViewerToolWindowControl.xaml]: I see what you mean.  I think it looks odd if the text is blue.

**Pull request merged**